Remotion/ObjectBinding/Web.Development.WebTesting.IntegrationTests/BocReferenceValueControlObjectTest.cs

```diff
@@ -223,8 +223,8 @@ public void TestGetOptionDefinitions ()
 
       Assert.That (options[15].ItemID, Is.EqualTo (fValue));
       Assert.That (options[15].Index, Is.EqualTo (16));
-      
-      if (Helper.BrowserConfiguration.IsChromium())
+
+      if (Helper.BrowserConfiguration.IsEdge())
```

Comment:
Hmm... with the double dependency on Chrome and Edge, this might be a back-and-forth change, isn't it?

Reply:
My plan would have been to first merge Chrome. Then, after rebasing the Edge branch, this `if` can be disposed of entirely.

This `IsEdge()` check is only still in place here because otherwise, nightly builds would fail for Edge (since it is not upgraded to 89 yet on this branch).

Reply:
ah, okay, that's fine.